package.json

```diff
@@ -57,7 +58,8 @@
 		"vue-server-renderer": "^2.5.7",
 		"vuex": "^3.0.1",
 		"vuex-router-sync": "^5.0.0",
-		"webpack-merge": "^4.1.2"
+		"webpack-merge": "^4.1.2",
+		"winston": "^2.4.2"
```

Comment:
Nice call using Winston, seems like it will help us out a lot: https://www.npmjs.com/package/express-winston